### ISSUE TITLE
Initializer_List naming confusion

### ISSUE THREAD

**Issue by cbaugher** [braydenanderson2014/C-Arduino-Libraries#63]
The Arduino IDE gets confused with the Initializer_List library because the actual .h file is named "Initilizer_list" with a small "l", and library.json reports it as "Initilizer_List" with an uppercase "L".

The Simple Vector library references it with a small "l".  Not sure about other libraries.

**Comment by braydenanderson2014** [comment]
Yeah I've noticed that yesterday all the sudden. I will see what can be done to fix that.

**Comment by braydenanderson2014** [comment]
What board are you using the library on... and you said you were using the arduino ide or vs code with platformio? I've noticed the issue is only with certain board definitions.

**Comment by braydenanderson2014** [comment]
Thank you... I renamed the file and posted it on the GitHub repository under the development branch. I will fix the issue with Simple Vector later. Due to the amount of libraries I have coded it is sometimes hard to remember to update documentation for libraries that depend on the one i edit.. im not even 100% certain that library even works correctly which is why on platformio its a alpha version. But thank you for reporting this issue so I can address it.
